template/{{app_name}}/src/app/[locale]/health/page.tsx

```diff
@@ -1,3 +1,37 @@
+'use client'
```

Comment:
(nit) This means the whole may be rendered in the client. Alternatively the members list could be a client component that the page uses so the page could still be rendered server side.